src/sentry/utils/suspect_resolutions_releases/get_suspect_resolutions_releases.py

```diff
@@ -0,0 +1,47 @@
+from datetime import timedelta
+from typing import Sequence
+
+from django.utils import timezone
+
+from sentry import features
+from sentry.models import Group, GroupStatus, Project, Release
+from sentry.signals import release_created
+from sentry.tasks.base import instrumented_task
+from sentry.utils.suspect_resolutions_releases import ALGO_VERSION, analytics
+
+
+@release_created.connect(weak=False)
+def record_suspect_resolutions_releases(release, project, **kwargs) -> None:
+    if features.has("projects:suspect-resolutions-releases", project):
+        get_suspect_resolutions_releases.delay(release, project)
+
+
+@instrumented_task(
+    name="sentry.tasks.get_suspect_resolutions_releases", queue="get_suspect_resolutions_releases"
+)
+def get_suspect_resolutions_releases(release: Release, project: Project) -> Sequence[int]:
+    suspect_resolution_issue_ids = []
+    is_suspect_resolution = False
+
+    suspect_issue_candidates = list(
+        Group.objects.filter(
+            status=GroupStatus.UNRESOLVED,
+            project=project,
+        )
+    )
+
+    for issue in suspect_issue_candidates:
+        if (timezone.now() - timedelta(days=7)) < issue.last_seen < release.date_released:
```

Comment:
Need to add some sort of a delay here - the `last_seen` will always be before `date_released` since this is run when a release is created, so `date_released` is the same as `timezone.now()`. Not sure what the best way to go about doing this is.

Reply:
@barkbarkimashark Also, instead of checking that the issue had events within the last 7 or `x` amount of days, maybe a better check would be to see if the issue had events in the release prior to the current release, and if it no longer has events after the current release. This way we can infer that this is the specific release that resulted in the resolution of the issue.